simplex.md

```diff
@@ -2,230 +2,84 @@
-
-While it is not required to be supported in the v1 of the protocol, it is
-important to have clarity on how all these scenarios can be supported in the
-future.
-
-## Chat system features
-
-- No user identity known to system servers - no phone numbers, user names and no
-  DNS are needed to identify the users to the system.
-- Each user can be connected to multiple servers to ensure message delivery,
-  even if some of the servers are compromised.
-- Uses standard asymmetric cryptographic protocols, so that system users can
-  create independent server and client implementations complying with the
-  protocols.
-- Open-source server implementations that can be easily deployed by any user
-  with minimal technical expertise (e.g. on Heroku via web UI).
-- Open-source client implementations so that system users can independently
-  assess system security model.
-- Only client applications store user profiles, contacts of other user profiles,
-  messages; servers do NOT have access to any of this information and (unless
-  compromised) do NOT store encrypted messages or any logs.
-- Multiple client applications and devices can be used by each user profile to
-  communicate and to share connections and message history - the devices are not
-  known to the servers.
-- Initial key exchange and establishing connections between user profiles is
-  done by sharing QR code via any independent communication channel (or directly
-  via screen and camera), system servers are NOT used for key exchange - to
-  reduce risk of key substitution in [MITM attack][2]. QR code contains the
-  connection-specific public key and other information needed to establish the
-  connection.
-- Connections between users can be established via shared trusted connections to
-  simplify key exchange.
-- Servers do NOT communicate with each other, they only communicate with client
-  applications.
+Existing chat platforms and protocols have some or all of the following problems:
+
+- Lack of privacy of the conversation, partially caused by [E2EE][1] implementations.
+- Lack of privacy of the user profile and connections.
+- Unsolicited messages (spam and abuse).
+- Lack of data ownership and protection.
+- Complexity of usage for all non-centralized protocols to non-technical users.
+
+The concentration of the communication in a small number of centralized platforms makes resolving these problems quite difficult.
+
+## Proposed solution
+
+Proposed stack of protocols solves these and other problems by making both messages and contacts accessible only on client devices, reducing the role of the servers to simple message brokers that only require authorization of messages sent to the queues, but do NOT require user authentication - not only the messages but also the metadata is protected.
+
+See [SMP protocol][6] and [SMP agent protocol][8].
+
+## Comparison with other protocols
+
+|          | SimpleX chat | Signal, big platforms | XMPP, Matrix | P2P protocols |
```

Comment:
shall we add comparison with [Session](https://getsession.org/)? - we did see it being talked about, and I think it might be good to acknowledge and highlight differences

Reply:
possibly later - need to formulate. It still has identity, and servers form a single network and communicate with each other. Also the commercial incentive ("crypto") is decoupled from message passing, which creates systemic risks for the whole network.